src/main/java/org/wildfly/security/auth/server/RealmIdentity.java

```diff
@@ -42,6 +42,11 @@
 public interface RealmIdentity {
 
     /**
+     * The timeout attribute for an identity.
+     */
+    String TIMEOUT_ATTRIBUTE = "timeout";
```

Comment:
Having a mechanism-specific attribute here seems wrong somehow.  On the other hand it could be argued that "updateable timeouts relating to mutual exclusion due to read-modify-write on successful auth" is a class mechanism rather than just a mechanism.  But in this case, timeout might be better expressed as a property of the identity itself, rather than an attribute to be found at some predefined name.